packages/devtools_app/lib/src/vm_service_wrapper.dart

```diff
@@ -841,9 +841,9 @@ class VmServiceWrapper implements VmService {
   }
 
   @override
-  Future<Success> setExceptionPauseMode(String isolateId, String mode) {
-    return trackFuture('setExceptionPauseMode',
-        _vmService.setExceptionPauseMode(isolateId, mode));
+  Future<Success> setIsolatePauseMode(String isolateId, String mode) {
```

Comment:
based on the bot failures, looks like the method signature needs to be changed to `Future<Success> Function(String, {String exceptionPauseMode, bool shouldPauseOnExit})`